themes/grass/layouts/partials/banner.html

```diff
@@ -1,5 +0,0 @@
-<div class="gitter-banner"
```

Comment:
would be good to also remove the css class in case is not used anymore

Reply:
I was not sure if we want to remove the banner completely, maybe we would want to highlight something else in the future? So I went with minimal changes...

Reply:
should be a quick fix (just some css declarations [here](https://github.com/OSGeo/grass-website/blob/master/themes/grass/assets/css/style.css#L676)). Issue being that css tends to accumulate dead code that at some point would need to be removed :)